google-cloud-pubsub/acceptance/pubsub/pubsub_test.rb

```diff
@@ -173,10 +173,12 @@ def retrieve_snapshot project, subscription, snapshot_name
       next_subs.token.must_be :nil?
     end
 
-    it "should allow creation of a subscription" do
-      subscription = topic.subscribe "#{$topic_prefix}-sub3"
+    it "should allow creation of a subscription with options" do
```

Comment:
After looking at the implementation more I don't think this is as important as I thought at first. The potential issue is like to avoid isn't very likely. Sometimes creating protobuf objects with nil values causes issues, but we aren't doing that here with Subscription. Up to you to cover that with acceptance tests or not.